.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
This file seems VERY small compared to the .eslintrc.js file. We should make sure that all rules still apply.

Reply:
This is a difficult one. I'm not an expert in ESLint. We have the choice jumping or just sticking with Next10 until someone takes the time to figure out what all the rules mean and how they compare to Next11's settings.

Reply:
Nor am I - but we shouldn't just _discard_ all the eslint rules we had before.
If an update to Next11 _requires_ us to use .json instead of .js, we should take the time to make sure the settings gets transferred (or least it with the .js if json is not required).

There should be some documentation or migration guide on this somewhere 🤔

Reply:
The reason it is so short is because, I simply created a fresh NextJS 11 app with TypeScript using `create-next-app` to see what the default was and that was all that was in there.  Perhaps we should find out how much we deviated fro NextJS 10 `create-next-app`.

Reply:
Ahh, I see.
I believe we had some custom eslint rules before, which would explain why the file is bigger.

We shouldn't reset the .eslint file - though as you said, we probably need to check up on the rules again in a seperate task, to ensure that all rules are needed.

Would it be possible to revert this file back to the original one we had, and then just add the new `react/no-unescaped-entities` rule there? .. I think that's the best approach for now.

Reply:
Next ESLint documentation: https://nextjs.org/docs/basic-features/eslint#eslint-config

Reply:
Hmm, though I think .eslint.js is still valid. It normally is .. I think .json is just the default 🤔

_If_ we switch to .json, we should make sure that the rules follows. We certainly don't want to discard any eslint rules we had in the old file, as they were custom rules.

[This pr](https://github.com/chingu-voyages/moonshot-chingu-quiz/pull/19/files) shows that the rules were custom for the project, so it's not from the initialization of the .eslint.js file :)